src/pages/includes/multiple-outputs/multiple-outputs.ts

```diff
@@ -28,9 +29,13 @@ export class MultipleOutputsPage {
     this.showMultiplesOutputs = false;
   }
 
+  @Output() openBlockChainEvent = new EventEmitter<string>();
   @Input()
   set tx(tx) {
     this._tx = tx;
+    this._misunderstoodOutputsMsg = tx.misunderstoodOutputs
+      ? 'There are some misunderstood outputs, please view on blockchain.'
```

Comment:
this.translate.instant('There are some misunderstood outputs, please view on blockchain.') here.

Reply:
Done! Thanks!